test/channel_test.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'test_helper.rb'
+
+describe "class Channel" do
+  before do
+    @channel = SlackCLI::Channel.new("seattle-stuff", "U786YWDVC3D", 71, "coffee shops")
+  end
+
+  it "creates an instance of channel" do
+    expect(@channel).must_be_kind_of SlackCLI::Channel
+  end
+
+  it "returns channel details correctly" do
+    expect(@channel.details).must_be_kind_of String
+  end
+
+  it "returns a list of channels" do
+    VCR.use_cassette "channels_list" do
+      response = SlackCLI::Channel.list_all
+      expect(response).must_be_kind_of Array
+    end
+  end
```

Comment:
It's true that a request-response cycle happens during the `SlackCLI::Channel.list_all` method call-- you're right! When we call `SlackCLI::Channel.list_all`, at some point, we make the API call with HTTParty in it.

However,  this line in this test reads a little weirdly. What gets returned from the `SlackCLI::Channel.list_all`? A list of channels gets returned from that method call, not a response!

It might read better overall to see this test as...

```suggestion
  it "returns a list of channels" do
    VCR.use_cassette "channels_list" do
      channels = SlackCLI::Channel.list_all
      expect(channels).must_be_kind_of Array
    end
  end
```

Especially when you read it with your well-written `it "returns a list of channels"` line!